doc/redirects.csv

```diff
@@ -7,13 +7,10 @@ release,whatsnew/index
 
 # getting started
 install,getting_started/install
-10min,getting_started/10min
```

Comment:
Not sure how to specify the redirects if they switch sections @jorisvandenbossche

Reply:
I think this can be `10min,user_guide/10min` ?

Reply:
Perfect that worked.